packages/preview/modern-cornell-notebooks/0.2.0/typst.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "modern-cornell-notebooks"
+version = "0.2.0"
+entrypoint = "template/cornell.typ"
+license = "MIT"
+description = "A Typst template for creating notebooks with cornell style."
```

Comment:
"A typst template" is redundant:

```
description = "Notebooks with Cornell style."
```

Reply:
Maybe this would be more correctly phrased as "Notebooks in the Cornell style", but I'm no English expert, so if it sounds bad to you feel free to ignore me.